src/index.js

```diff
@@ -5,7 +5,9 @@ const pluginName = 'RetryChunkLoadPlugin';
 
 class RetryChunkLoadPlugin {
   constructor(options = {}) {
-    this.options = Object.assign({}, options);
+    this.options = Object.assign({
+      timeout: 3000,
```

Comment:
Please can we just leave this as `this.options = Object.assign({}, options);`, if we default it to 3000 it would technically be a breaking change, I think we should just leave it up to the user to decide